app/models/google/calendar/calendar.rb

```diff
@@ -129,8 +140,24 @@ def summary
       event_hash['summary']
     end
 
+    def description
+      event_hash['description']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.